blimflix/client/app/directives/topbar/topbar.directive.html

```diff
@@ -0,0 +1,27 @@
+<nav class="navbar navbar-inverse navbar-fixed-top">
+  <div class="col-lg-3 header-size">
+    <div class="navbar-header navbar-size">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="blimflix-logo-link" href="#">
```

Comment:
Please use the [ng-href directive](https://docs.angularjs.org/api/ng/directive/ngHref) and link to applicaton's **/** route.